soc/silabs/silabs_siwx91x/siwg917/nwp.c

```diff
@@ -255,6 +256,14 @@ static int siwg917_nwp_init(void)
 	}
 
 	if (IS_ENABLED(CONFIG_SOC_SIWX91X_PM_BACKEND_PMGR)) {
+		/*power save*/
+		status = sl_si91x_bt_set_performance_profile((	sl_bt_performance_profile_t*)&performance_profile);
```

Comment:
I wonder what happens if BT is disabled (either with `status = "disabled"` or with `CONFIG_BT_SILABS_SIWX91X=n`). Is it the right place for this call? (note *I believe* the things are different for `sl_wifi_set_performance_profile()` because it controls the whole NWP)